### PR TITLE
Disabled deleting own account

### PR DESCRIPTION
**Pull request by rkusan00**
- removed option to delete account of the current user in admin menu
- added tooltip for the user info

### CODE REVIEW

File: client/admin/components/common/Navbar.vue [hunk 1]
```diff
@@ -30,7 +33,12 @@ export default {
   props: {
     drawer: { type: Boolean, default: true }
   },
-  computed: mapState('auth', ['user']),
+  computed: {
+    ...mapState('auth', ['user']),
+    fullName() {
+      return this.user.firstName + ' ' + this.user.lastName;
```
Comment (zcuric) [inline client/admin/components/common/Navbar.vue]: You can use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) here. 
```js
return `${this.user.firstName} ${this.user.lastName}`
```
Maybe even use [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring)
```js
const { firstName, lastName } = this.user;
return `${firstName} ${lastName}`;
```

File: client/admin/components/users/index.vue [hunk 2]
```diff
@@ -100,7 +108,8 @@ export default {
   },
   computed: {
     headers,
-    defaultPage
+    defaultPage,
+    ...mapState('auth', ['user'])
```
Comment (zcuric) [inline client/admin/components/users/index.vue]: Move `...mapState('auth', ['user'])` at top of the object, so you don't accidentally override previously computed props.

File: client/admin/components/common/Navbar.vue [hunk 3]
```diff
@@ -30,7 +33,13 @@ export default {
   props: {
     drawer: { type: Boolean, default: true }
   },
-  computed: mapState('auth', ['user']),
+  computed: {
+    ...mapState('auth', ['user']),
+    fullName() {
+      const { firstName, lastName } = this.user;
+      return '${firstName} ${lastName}';s
```
Comment (vladimyr) [inline client/admin/components/common/Navbar.vue]: Hm, you have typo here :bowtie:

File: client/admin/components/users/index.vue [hunk 4]
```diff
@@ -43,7 +43,14 @@
...
                   mdi-pencil
                 </v-icon>
-                <v-icon @click="removeUser(props.item)" small class="ml-2">
+                <v-icon
+                  v-if="user.id != props.item.id"
+                  @click="removeUser(props.item)"
+                  small
+                  class="ml-2">
+                  mdi-delete
+                </v-icon>
```
Comment (vladimyr) [inline client/admin/components/users/index.vue]: Don't remove icon completely, fade it away and disable click handler :bowtie:

**Review by vladimyr — CHANGES_REQUESTED** [review]

File: client/admin/components/users/index.vue [hunk 5]
```diff
@@ -43,7 +43,11 @@
                 <v-icon @click="showUserDialog(props.item)" small>
                   mdi-pencil
                 </v-icon>
-                <v-icon @click="removeUser(props.item)" small class="ml-2">
+                <v-icon
+                  @click="removeUser(props.item)"
+                  :disabled="user.id == props.item.id"
```
Comment (vladimyr) [inline client/admin/components/users/index.vue]: Triple equals please :bowtie:

File: client/admin/components/users/index.vue (hunk 4 above)
Comment (vladimyr) [reply client/admin/components/users/index.vue]: Does setting `disabled` disables click handler too?

**Review by vladimyr — CHANGES_REQUESTED** [review]

**Comment by vladimyr** [comment]
@kjuej Please verify this one.

**Comment by zcuric** [comment]
@kjuej ☝️

**Pull request merged**